src/components/html/HTML.js

```diff
@@ -31,7 +31,16 @@ export default class HTMLComponent extends BaseComponent {
 
   setHTML() {
     this.htmlElement.innerHTML = this.interpolate(this.component.content);
-  }
+    const walker = document.createTreeWalker(this.htmlElement, NodeFilter.SHOW_TEXT, null, false);
```

Comment:
We should probably make this a configuration since I know this would probably kill the performance of some forms with very large content regions. We should call the config for this ```translateContent```

Reply:
@travist  how do this ?

Reply:
Add a new configuration setting within the HTML edit form seen here... 

https://github.com/formio/formio.js/blob/master/src/components/html/editForm/HTML.edit.display.js

Then you can reference that configuration within here using 

```
if (this.component.translateHtml) {
```

You will also want to add that property with the default to the schema at https://github.com/formio/formio.js/blob/master/src/components/html/HTML.js#L6

It should be defaulted to ```false```

Reply:
Hi travist,
 i added config as select component has true and false options. please check below : 
https://github.com/AnasHijazi/formio.js/invitations